app/src/main/java/com/github/sdp/tarjetakuna/database/UserRTDB.kt

```diff
@@ -0,0 +1,141 @@
+package com.github.sdp.tarjetakuna.database
+
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseReference
+import java.util.concurrent.CompletableFuture
+
+/**
+ * This class is used to manage the user's data in the database.
+ */
+class UserRTDB(db: DatabaseReference) { //Firebase.database.reference.child("users") //assumption: we check add ability in User
+
+    private var db: DatabaseReference
+    private val cardsRTDB = CardsRTDB(db.child("cards"))
+
+    init {
+        this.db = db
+    }
+
+    /**
+     * Adds a card to the users collection.
+     */
+    fun addCard(card: DBMagicCard, userUID: String) {
+        val cardUID = card.code + card.number
+        val fbpossession = card.possession.toString().lowercase()
+        db.child(userUID).child(fbpossession).child(cardUID).setValue(1)

```

Comment:
@Bjolidon you can change here later to increase the value by one for owning multiple time the same card